syntax = "proto3";

package PbTPMS;

message Rtl433Payload
{
    string time         = 1;
    string model        = 2;
    int32  id           = 3;
    double pressure     = 4;
    uint32 temperature  = 5;
    uint32 acceleration = 6;
    uint32 battery      = 7;
    uint32 interframe   = 8;
    uint32 wo_state     = 9;
    uint32 checksum     = 10;
    string mic          = 11;
}